src/main/java/net/mtrop/doom/tools/WTexScanMain.java

```diff
@@ -96,6 +99,7 @@ private Options()
 			this.outputFlats = false;
 			this.skipSkies = false;
 			this.wadFiles = new LinkedList<>();
+			this.mapsToScan = new LinkedList<>();
```

Comment:
Related to the previous comment (`List` to `Set`), implementation should be a `new TreeMap<>(String.CASE_INSENSITIVE_ORDER)` as the map headers are going to be uppercase (99.9% percent of the time) and command-line users shouldn't need to be forced to know the case as a "quality of life" feature.

Reply:
Correction: `TreeSet`, not Map